src/quote/Quote.ts

```diff
@@ -19,6 +19,16 @@ import Identity from '../identity/Identity'
 import { IQuote, IQuoteAgreement, IQuoteAttesterSigned } from '../types/Quote'
 import { hashObjectAsStr, verify } from '../crypto/Crypto'
 
+/**
+ * Validates the quote against the meta schema and quote data against the provided schema.
+ *
+ * @param schema A [[Quote]] schema object.
+ * @param validate [[Quote]] data to be validated against the provided schema.
+ * @param messages Errors provide an array of errors, in the case of validation failures.
```

Comment:
Not really changed as discussed ☺️ 
Can you please check again?
https://github.com/KILTprotocol/sdk-js/pull/234#discussion_r378294636